pkg/cmd/pr/merge/merge_test.go

```diff
@@ -926,7 +928,7 @@ func TestPrMerge_alreadyMerged_nonInteractive(t *testing.T) {
 			ID:                  "THE-ID",
 			Number:              4,
 			State:               "MERGED",
-			HeadRepositoryOwner: api.Owner{Login: "monalisa"},
+			HeadRepositoryOwner: api.Owner{Login: "OWNER"},
```

Comment:
this test was set up as a cross repo test due to the different owner of the repo and the pr. I modified it to match the intent of the test, and added a new one for the original condition `TestPrMerge_alreadyMerged_nonInteractive_crossRepo`